starter-code/src/viking.js

```diff
@@ -1,11 +1,93 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+  this.receiveDamage = function(damage) {
+    this.health = this.health - damage;
+  };
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
 
 // Viking
```

Comment:
Aquí está el primer error que te da Jasmine, En la iteración te indica que receiveDamage debe ser un Prototype por lo que no está pasado como tal. 
A continuación te dejo un el fragmento de código de esta iteración para mostrarte visualmente como está escrito ese prototype. 

function Viking (nameArg, healthArg, strengthArg) {
  Soldier.call(this, healthArg, strengthArg);
  this.name = nameArg;
}

Viking.prototype = Object.create(Soldier.prototype);
Viking.prototype.constructor = Viking;

Viking.prototype.receiveDamage = function (damage) {
  this.health -= damage;

  if (this.health > 0) {
    return this.name + " has received " + damage + " points of damage";
  }
  else {
    return this.name + " has died in act of combat";
  }
};

Viking.prototype.battleCry = function () {
  return "Odin Owns You All!";
};